java/client/src/test/java/glide/managers/ConnectionManagerTest.java

```diff
@@ -64,6 +64,8 @@ public class ConnectionManagerTest {
 
     private static final String CLIENT_NAME = "ClientName";
 
+    private static final int INFLIGHT_REQUESTS_LIMIT = 1000;
```

Comment:
why dont you put a brief comments why 1K and not other?

Reply:
That's a test, no need for desc here.
In the glide core it's all well doced.